lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
noop query works without imageTypeIfUnknown. If the image has a file extension it uses it to get content-type otherwise it just ignores the noop query if imageTypeIfUnknown is not set.
Are you sure you want I remove?
imageTypeIfUnknown seems hacky, I am not a fan too. But have have millions of files this way. Others may have too.
One next step could be read the image file header it imageTypeIfUnknown. This way noop could be fully functional anyway, even without file extension. I known JPEG is easy to check. I need to verify the others formats. I can work on it in a next step.
Your call. Let me know.